nf_core/launch.py

```diff
@@ -445,8 +435,8 @@ def prompt_group(self, group_id, group_obj):
         answers = {}
         while not while_break:
             self.print_param_header(group_id, group_obj)
-            answer = PyInquirer.prompt([question])
-            # TODO: use raise_keyboard_interrupt=True when PyInquirer 1.0.3 is released
+            answer = questionary.prompt([question])
+            # Raise keyboard interrupg
```

Comment:
Can hopefully get ride of this, but until then - typo..

```suggestion
            # Raise KeyboardInterrupt exception
```

Reply:
Can't comment on this line in the PR interface as you haven't edited it, but would be fab if you could add two hypens to the log message [below](https://github.com/nf-core/tools/pull/799/files#diff-31477effcfb136e692dd57c7d0abf196e9cb8962d49c1bb43cee0f2a7ad8c93bR449)

```diff
+ log.error("'--{}' is required.".format(p_required))
- log.error("'{}' is required.".format(p_required))
```

This should make the formatting consistent with the other identical message check.